QuizApp/QuizApp/Utils/SecurityStorage.swift

```diff
@@ -21,4 +23,8 @@ class SecurityStorage: SecurityStorageProtocol {
         keychain[accessTokenKey] = accessToken
     }
 
+    func clear() {
+        keychain[accessTokenKey] = nil
```

Comment:
The function is called clear but it only clears the keychain.

**NOTE**: You should never clear the whole keychain ;)